app/controllers/api/v1/doctors_controller.rb

```diff
@@ -17,24 +17,19 @@ def create
     @doctor = Doctor.new(doctor_params)
 
     if @doctor.save
-      render json: @doctor, status: :created, location: @doctor
-    else
-      render json: @doctor.errors, status: :unprocessable_entity
-    end
-  end
-
-  # PATCH/PUT /doctors/1
-  def update
-    if @doctor.update(doctor_params)
-      render json: @doctor
+      render json: 'Doctor successfully created✅', status: :created
     else
       render json: @doctor.errors, status: :unprocessable_entity
     end
   end
 
   # DELETE /doctors/1
   def destroy
-    @doctor.destroy
+    if @doctor.destroy
+      render json: 'Doctor successfully deleted✅ ', status: :created
+    else
+      render json: @appointment.errors, status: :unprocessable_entity
+    end
```

Comment:
[REQUIRED] - On line 31, You make a mistake you should use `render json: @doctor.errors, status: :unprocessable_entity` to get the right error message.